### DISCUSSION TITLE
Add authentication header to thumbnail requests

### DISCUSSION THREAD

**Discussion by timpie90** [ProjectMirador/mirador#3507] [Q&A]
Hello,

I have a question about the thumbnails in the Mirador Viewer. I have an image server with auhtentication enabled. I got Mirador to send an auth header with the request for the manifest and it sends an auth header for every request after that. 

The problem I have is with the thumbnails, I wonder how I can send an auth header for those requests as well?

Thanks in advance!

**Comment by tunztunztunz** [comment]
I know this question is old, but is there a known solution for this? Setting auth in isdConfig's `ajaxHeaders` seems to work for the IIIF image and using a preprocessors array in `requests` works for the info.json fetch, but thumbnails continue to throw 401 auth errors.

**Comment by marlo-longley** [comment]
This got lost in the discussion section. Moved to an issue: https://github.com/ProjectMirador/mirador/issues/4519
Thank you both.
